src/Game.cpp

```diff
@@ -250,13 +251,20 @@ bool Game::ProcessEvents(const SDL_Event& event)
 		case SDLK_8:
 			if ((event.key.keysym.mod & KMOD_CTRL) != 0)
 			{
-				auto handPosition = _handPose * glm::vec4(0.0f, 0.0f, 0.0f, 1.0f);
+				const auto handPosition = _handPose * glm::vec4(0.0f, 0.0f, 0.0f, 1.0f);
 				const auto index = static_cast<uint8_t>(event.key.keysym.sym - SDLK_1);
-				Locator::cameraBookmarkSystem::value().SetBookmark(index, handPosition);
+				Locator::cameraBookmarkSystem::value().SetBookmark(index, handPosition, _camera->GetOrigin());
 			}
 			else if (event.key.keysym.mod == 0)
```

Comment:
```suggestion
			else
```
This check should probably be removed as it blocks restoring bookmarks while num lock is active